features/simple_format.feature

```diff
@@ -176,6 +176,13 @@ Feature: Showing build output in simple format
         Given podfile.lock wasn't in sync
         When I pipe to xcpretty with "--simple --color"
         Then I should see a red error message
+        And I should see that sandbox is not in sync with Podfile.lock
+
+    Scenario: Compilation fails because missing files in the project
+        Given there was a missing file
+        When I pipe to xcpretty with "--simple --color"
+        Then I should see a red error message
+        #And I should see which file is missing
```

Comment:
Oops, good catch!

--
Marin Usalj
supermar.in

On Tue, Oct 13, 2015 at 3:45 PM, Delisa Mason <notifications@github.com>
wrote:

> In features/simple_format.feature
> <https://github.com/supermarin/xcpretty/pull/178#discussion_r41936292>:
>
> > @@ -176,6 +176,13 @@ Feature: Showing build output in simple format
> >          Given podfile.lock wasn't in sync
> >          When I pipe to xcpretty with "--simple --color"
> >          Then I should see a red error message
> > +        And I should see that sandbox is not in sync with Podfile.lock
> > +
> > +    Scenario: Compilation fails because missing files in the project
> > +        Given there was a missing file
> > +        When I pipe to xcpretty with "--simple --color"
> > +        Then I should see a red error message
> > +        #And I should see which file is missing
>
> What's going on here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/supermarin/xcpretty/pull/178/files#r41936292>.
>